packages/office-ui-fabric-react/src/components/ScreenReaderAlert/ScreenReaderAlertPage.tsx

```diff
@@ -0,0 +1,121 @@
+              code={ ScreenReaderAlertRepeatExampleCode as string }
+            >
+              <ScreenReaderAlertRepeatExample />
+            </ExampleCard>
+            <ExampleCard
+              title='ScreenReaderAlert with Spinner'
+              code={ ScreenReaderAlertSpinnerExampleCode as string }
+            >
+              <ScreenReaderAlertSpinnerExample />
+            </ExampleCard>
+            <ExampleCard
+              title='ScreenReaderAlert polite mode'
+              code={ ScreenReaderAlertAfterExampleCode as string }
+            >
+              <ScreenReaderAlertAfterExample />
+            </ExampleCard>
+          </div>
+        }
+        propertiesTables={
+          <PropertiesTableSet
+            sources={ [
+              require<string>('!raw-loader!office-ui-fabric-react/src/components/ScreenReaderAlert/ScreenReaderAlert.Props.ts')
+            ] }
+          />
+        }
+        overview={
+          <div>
+            <p>
+              ScreenReaderAlerts are used to read out the notifications on the page to the blind people with the help of screen readers like JAWS, Chrome VOX and NVDA. It is implemented by live region and overcomes some known issues about live region when using with React.
+            </p>
+            <p>
+              It renders an addition element for screen reader only, so using this component while the text/ui is visible will duplicate the text in the DOM. You should use vanilla live region attributes instead if you don't want this duplicate element.
+            </p>
+            <p>
+              This component has been tested through following combination of screen readers and browsers:
+            </p>
+            <ul>
+              <li>
+                1. <strong>Narrator in windows 10 Enterprise 14393.577</strong> with <strong>Edge 38.14393.0.0</strong>.
+              </li>
+              <li>
+                2. <strong>NVDA 2016.4</strong> with <strong>FireFox v50.1.0</strong>.
+              </li>
+              <li>
+                3. <strong>Chrome VOX v53</strong> with <strong>Chrome v54</strong>.
+                <br />
+                Note: ReadingMode.ReadAfterOtherContent option is currently not supported in ChromeVOX, text will be read immediately.
+              </li>
+              <li>
+                4. <strong>JAWS 18</strong> with <strong>IE 11</strong>.
+              </li>
+            </ul>
+
+            <p>
+              To try the demo in this page, you should at least using one kind of screen reader. E.g. JAWS, ChromeVOX, NVDA.
+              Better use the browser recommended by the screen reader you're using. E.g. IE11 with JAWS, Chrome with ChromeVOX, FireFox with NVDA.
+            </p>
+          </div>
+        }
+        bestPractices={
+          <div></div>
+        }
+        dos={
+          <div>
+            <ul>
+              <li>Use if it would be hard for blind people to know dynamic changes on the page.</li>
```

Comment:
@dzearing We don't need auto clear text in case of spinners. @ipip2005, I see a lot of value in having a well tested component that can be plugged anywhere to provide text announcements. But when it comes to attaching it to existing components, I share the concern as David:

1. It will add additional payload with a simpler component like spinner and some of the value add features (auto clear. atomic updates) may not be relevant to all components.

2.  Most importantly, a lot of fabric consumers are not aware of accessibility world (screen readers, aria patterns and behaviors) and they expect things to work out of box. As a consumer if I take a dependency on Spinner, I would expect it to take care of accessibility support. Looking at the examples here, we need to add this component explicitly along with other components to make them accessibility which feels like an anti-pattern. 

The other pattern of aria-live + DelayedRender has been working nice for MessageBar, TextField validation etc.

**Proposal**: We can keep the ScreenReaderAlert as an exposed component for those who need more fine-grained control over announcements but individual components should have their own accessibility taken care internally.